tests/main/upgrade-from-release/task.yaml

```diff
@@ -44,6 +48,7 @@ execute: |
     fi
 
     declare -A EXPECTED_SNAPD_VERSIONS=(
+        ["24.04"]='2.60.4\+24.04'
         ["23.04"]='2.59.1\+23.04'
         ["22.04"]='2.55.3\+22.04'
```

Comment:
Should there be one for mantic as well? Why just lunar?

Reply:
fixed, mantic added.